epub33/rs/index.html

```diff
@@ -450,11 +450,6 @@ <h4>Metadata</h4>
 			<section id="sec-pkg-doc-manifest">
 				<h3>Manifest</h3>
 
-				<p id="confreq-rs-pkg-manifest-url" data-tests="#pkg-manifest-url,#pkg-relative-url">When an
-						<code>href</code> attribute contains a <a data-cite="url#relative-url-string">relative-URL
-						string</a>, Reading Systems MUST use the URL of the Package Document as the <a
-						data-cite="url#concept-base-url">base URL</a> when <a data-cite="url#concept-url-parser"
-						>parsing</a> to a <a data-cite="url#concept-url">URL record</a> [[URL]].</p>
 
```

Comment:
Should https://w3c.github.io/epub-specs/epub33/rs/#issue-container-generatedID-12 now be MUST instead of must? It seems strange to me that there would be no normative requirement for RSes to parse base URLs correctly.

Reply:
I believe the same argument holds as in https://github.com/w3c/epub-specs/issues/1918#issuecomment-985485675 of @rdeltour: we should try to avoid redundant MUST-s. The note refers to the relevant [MUST statements in the core spec](https://w3c.github.io/epub-specs/epub33/core/#sec-parsing-urls-metainf).

A certain level of redundancy between the RS and the Core are unavoidable: these two specs are very closely related. (One could argue that the whole [section](https://w3c.github.io/epub-specs/epub33/rs/#sec-container-iri) in the RS is redundant as the constraints at the start of the section are the same as in the [content spec](https://w3c.github.io/epub-specs/epub33/core/#sec-container-iri), but sometimes the "readability" of the spec is also at stake (the whole construction of the root URL is crucial to the spec), so it is all right in my view.